components/MemoryCard/MemoryCardDetail.js

```diff
@@ -23,18 +23,8 @@ export default function MemoryCardDetail({ currentEvent, onToggleFavorite }) {
             id={currentEvent.id}
           />
           <p>date: {formatDate}</p>
-          <h3>{currentEvent.name}</h3>
-          <p>
-            Lorem Ipsum is simply dummy text of the printing and typesetting
-            industry. Lorem Ipsum has been the industry's standard dummy text
-            ever since the 1500s, when an unknown printer took a galley of type
-            and scrambled it to make a type specimen book. It has survived not
-            only five centuries, but also the leap into electronic typesetting,
-            remaining essentially unchanged. It was popularised in the 1960s
-            with the release of Letraset sheets containing Lorem Ipsum passages,
-            and more recently with desktop publishing software like Aldus
-            PageMaker including versions of Lorem Ipsum.
-          </p>
+          <h3>{currentEvent.headline}</h3>
+          <p>{currentEvent.details}</p>
         </StyledTextFormatter>
       </StyledDetailFrame>
     </>
```

Comment:
Den Wrapper <> kannst du in diesem Fall entfernen, da dein article bereits den Inhalt umfasst.

Reply:
good point 👍 thx